src/orquestra/sdk/_base/_conversions/_invocations.py

```diff
@@ -311,6 +314,14 @@ def _make_constant_input(
     }
 
 
+def _make_secret_input(arg_name: str, secret: ir.SecretNode) -> yaml_model.StepInput:
+    json_dict = json.loads(secret.json())
```

Comment:
**thought**: it would be nice to have a single place where we transform a Pydantic to a JSON-compatible dict instead of `json.loads(model.json())` all over the place. 

For example, is there a reason to include `null`s in the output YAML? The default behavior of `BaseModel.json()` is to include `None`s

Reply:
I added a helper for this. I left the nulls for now, because when we export to YAML we handle this. f8f309a